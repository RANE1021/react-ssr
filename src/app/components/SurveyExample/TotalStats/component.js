import React from "react";

//need to pass in styles and events
const Stats = ({ yes, no , total, styles }) => {
  return (
    <div>
      <div>Total Votes:{total}</div>
      <div>Yes:{yes}</div>
      <div>No:{no}</div>
    </div>
  );
};

export default Stats;
