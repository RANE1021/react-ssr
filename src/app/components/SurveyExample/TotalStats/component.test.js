import React from "react";
import { shallow } from "enzyme";

import TotalStats from "./component";

const styles = {};

describe("The home component mounts with styles and four divs", () => {
  const shallowTotalStats = shallow(<TotalStats styles={styles} />);
  it("loads with divs", () => {
    expect(shallowTotalStats.find("div").length).toBe(4);
  });

  it('matches previous snapshot', () => {
    expect(TotalStats).toMatchSnapshot()
  })
});
