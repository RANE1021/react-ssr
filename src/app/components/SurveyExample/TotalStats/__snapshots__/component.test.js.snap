// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The home component mounts with styles and four divs matches previous snapshot 1`] = `[Function]`;

exports[`The home component mounts with styles and three divs matches previous snapshot 1`] = `[Function]`;
